categories/communication/rules-to-better-communication.md

```diff
@@ -30,6 +30,7 @@ index:
   - tofu
   - attention-to-detail
   - collaborate-across-timezones
+  - what-to-do-when-a-client-says-no
```

Comment:
Hi @RobJT-wq 

We have a category for dealing with clients... which I believe is a better fit for this new rule

- [ ] 1. Can you review and fix
- [ ] 2. Consider adding it next to a similar rule, instead of straight at the end